06.bowling_object/frame.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require './shot'
+
+class Frame
+  attr_reader :shots
+
+  def initialize(shots)
+    @shots = shots
+  end
+
+  def scores
+    @shots.sum(&:score)
+  end
+
+  def strike?
+    @shots[0].score == 10
+  end
+
+  def spare?
+    @shots[0].score + @shots[1].score == 10
```

Comment:
```suggestion
    @shots[0..1].sum(&:score) == 10
```

こんな書き方でも良さそう。

あと念のため「strikeでないこと」も条件に含めるといいかも